libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -86,11 +87,12 @@ CK_SESSION_HANDLE xGlobalSession = 0;
 CK_FUNCTION_LIST_PTR pxGlobalFunctionList = NULL_PTR;
 CK_SLOT_ID xGlobalSlotId = 0;
 CK_MECHANISM_TYPE xMechanismType = 0;
-CK_OBJECT_HANDLE xPublicKey = 0;
-CK_OBJECT_HANDLE xPrivateKey = 0;
-CK_OBJECT_HANDLE xKey = 0;
-CK_BBOOL xCkTrue = CK_TRUE;
-CK_BBOOL xCkFalse = CK_FALSE;
+
+/* MBT test group variables. */
```

Comment:
What is MBT ? Can we write complete name instead of acronym?

Reply:
It's the Model Based Tests, they were machine generated based on the PKCS #11 specification and contributed to us. I'll update this to the full name